CHANGELOG.md

```diff
@@ -2,6 +2,54 @@
 
 ## unreleased
 
+### Breaking
+* Removed Algob prefix in deployer (eg. renamed `AlgobDeployer` to `Deployer`)
+* Updated `execParams` structure & typings (input parameters for `executeTransaction`)
+    * Migration: If `SignType` is `LogicSignature` then change `fromAccount` to `fromAccountAddr` and just pass from address instead of complete account.
+* [breaking] Changed the way we pass arguments to stateless smart contract - moved assignment from when we load smart contract (using `loadLogic`, `mkDelegatedLsig`, `fundLsig`) to when we create transaction execution parameters.
+    * Migration: assign stateless args in txParams to `executeTransactin`. Eg
+        ```js
+        await deployer.loadLogic('htlc.py', [arg1]); // remove scTmplParams from here
+        const txnParams: rtypes.AlgoTransferParam = { .. }
+        txnParams.args = [arg1]; // assign here now
+        await executeTransaction(deployer, txnParams);
+        ```
+
+### Improvements
+* Added more tests for the [crowdfunding example project](/examples/crowdfunding) using `@algo-builder/runtime`- Happy paths and Failing paths.
+* Added new command `algob test` (runs mocha in project root).
```

Comment:
We can relocate this to the subsection under Line 23.

Reply:
yes, thanks for suggestion!